db/seeds.rb

```diff
@@ -42,8 +43,8 @@ def random_organization
   ##Organizations
   orgs = YAML::load_file(Rails.root.join("db/seed_files/orgs.yml"))
 
-  ## NOTE: Org type is being randomly assigned for now until we get a specific mapping.
-  orgs.each{ |org| Organization.create_with(org_type: Organization::org_types.keys.sample).find_or_create_by!(name: org[:name], fb_url: org[:fb_url]) }
+  ## NOTE: Org type, avatar_url is being randomly assigned and location hard coded for now until we get a specific mapping.
+  orgs.each{ |org| Organization.create_with(org_type: Organization::org_types.keys.sample).find_or_create_by!(name: org[:name], fb_url: org[:fb_url], avatar_url: Faker::Avatar.image(nil,"50x50", "jpg", "any", "any"), location: "#{Faker::Address.city}, IL") }
```

Comment:
@gcwokeup it looks like `rake db:seed` is failing on this line. Is there a migration missing for location field on Organization?

Reply:
im adding it. i think it got lost on a bad merge.